platform/mellanox/fw.mk

```diff
@@ -1,16 +1,32 @@
 # mellanox firmware
 
-MLNX_FW_BASE_URL = $(MLNX_SDK_BASE_URL)
+MLNX_FW_BASE_PATH = $(MLNX_SDK_BASE_PATH)
+
+# Place an URL here to FW if you want to download FW instead
+MLNX_FW_BASE_URL =
```

Comment:
Would it be better to initialize it to some default value (e.g. empty string / "None", ...) ?

Reply:
Basically it is an empty string. The issue with "None" is that it could be a valid input for wget (which is used internally in slave.mk)

```
stepanb@arc-build-server:~$ wget none/fw-SPC1.mfa
--2019-10-07 17:49:56--  http://none/fw-SPC1.mfa
Resolving none (none)... failed: Name or service not known.
wget: unable to resolve host address ‘none’

```

I doubt that someone will call a server as "none" and will rely on wget default protocol http, but in that case they won't be able to use it without specifying protocol.